run-tests.pl

```diff
@@ -371,12 +371,17 @@ sub log_if_fail
    push @log_if_fail_lines, split m/\n/, pp( $structure ) if @_ > 1;
 }
 
-struct Fixture => [qw( requires start result teardown )], predicate => "is_Fixture";
+struct Fixture => [qw( name requires start result teardown )], predicate => "is_Fixture";
 
+my $fixture_count = 0;
 sub fixture
 {
    my %args = @_;
 
+   # make up an id for later labelling etc
+   my $count = $fixture_count++;
+   my $name = $args{name} || "FIXTURE-$count";
```

Comment:
Generally best to use `//` which is the defined-or operator, in case someone did actually want to set something like an empty string or zero as a name

Reply:
indeed. 973c87c